### PR TITLE
CLN Replace old string formatting syntax with f-strings #29547: 2 files w…

### PR DESCRIPTION
**Pull request by OlivierLuG**
Hello, i'm new to github / opensource. Let me know if I'm doing things correctly

pandas/_libs/tslibs/timedeltas.pyx
pandas/_libs/tslibs/timestamps.pyx

Replace old string formatting syntax with f-strings #29547

- [ x ] passes `black pandas`
- [  ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -584,7 +584,7 @@ cdef inline int64_t parse_iso_format_string(str ts) except? -1:
         bint have_dot = 0, have_value = 0, neg = 0
         list number = [], unit = []
 
-    err_msg = "Invalid ISO 8601 Duration format - {}".format(ts)
+    err_msg = "Invalid ISO 8601 Duration format - {ts}"
```
Comment (dsaxton) [inline pandas/_libs/tslibs/timedeltas.pyx:587]: Missing the f

File: pandas/_libs/tslibs/timestamps.pyx [hunk 2]
```diff
@@ -194,7 +194,7 @@ def integer_op_not_supported(obj):
         "Addition/subtraction of integers and integer-arrays with {cls} is "
         "no longer supported.  Instead of adding/subtracting `n`, "
         "use `n * obj.freq`"
-    ).format(cls=cls)
+    )
```
Comment (dsaxton) [inline pandas/_libs/tslibs/timestamps.pyx:197]: Here as well

File: pandas/_libs/tslibs/timedeltas.pyx (hunk 1 above)
Comment (OlivierLuG) [reply pandas/_libs/tslibs/timedeltas.pyx:587]: How dumb I am !  I will propose a new (valid) PR tomorow.
Thank for the reply

**Comment by OlivierLuG** [comment]
Hello, I’ve fixed the 2 issues. All test were passed successfully.

**Comment by jreback** [comment]
thanks @OlivierLuG

**Pull request merged**